pkg/actors/actors.go

```diff
@@ -979,9 +979,6 @@ func (a *actorsRuntime) startReminder(reminder *reminders.Reminder, stopChannel
 				break L
 			}
 
-			if nextTimer.Stop() {
```

Comment:
Reviewers: this was deleted because when we hit this stage, the reminder has already fired. Trying to drain it again would cause a deadlock.